lib/camerata.rb

```diff
@@ -129,6 +129,29 @@ def console(service, *args)
     end
     map rc: :console
 
+    desc "set_env TARGET_NS SOURCE_NS", "copy params from env to another"
+    def env_copy(target_ns, source_ns = "")
+      app_versions = Camerata::AppVersions.get_all source_ns
+      secrets = Camerata::Secrets.get_all source_ns
+      puts "\n Copying following parameters from source to #{target_ns} namespace: " \
+           "\n APP VERSIONS: #{app_versions}" \
+           "\n SECRETS: #{secrets}"
+
+      app_versions.each { |app, version| Camerata::Parameters.set("#{target_ns}_#{app}", version) }
+      secrets.each { |app, version| Camerata::Parameters.set("#{target_ns}_#{app}", version, true) }
+    end
+
+    desc "get_env KEY", "get details of a parameter"
+    def env_get(key)
```

Comment:
Add error-checking -- if I put in a nonsense key, result is null and I get a stack trace back.  Looking good for things that exist though!  
![image](https://user-images.githubusercontent.com/21695512/91494780-628f2a00-e887-11ea-9e2a-617a010c8000.png)

Reply:
![image](https://user-images.githubusercontent.com/21695512/91888645-a359bb00-ec5a-11ea-849b-7ea78f6eab0f.png)

Yay!